app/src/main/java/org/fossasia/pslab/activity/LuxMeterActivity.java

```diff
@@ -0,0 +1,114 @@
+
+                        switch (item.getItemId()) {
+                            case R.id.action_data:
+                                if (!(fragment instanceof LuxMeterFragmentData))
+                                    selectedFragment = LuxMeterFragmentData.newInstance();
+                                break;
+                            case R.id.action_config:
+                                if (!(fragment instanceof LuxMeterFragmentConfig))
+                                    //
+                                break;
+                            default:
+                                break;
+                        }
+                        if (selectedFragment != null) {
+                            FragmentTransaction transaction = getSupportFragmentManager().beginTransaction();
+                            transaction.replace(R.id.frame_layout_lux_meter, selectedFragment);
+                            transaction.commit();
+                        }
+                        return true;
+                    }
+                });
+        howToConnectDialog(getString(R.string.lux_meter), getString(R.string.lux_meter_intro), R.drawable.bh1750_schematic, getString(R.string.lux_meter_desc));
+        try {
+            FragmentTransaction transaction = getSupportFragmentManager().beginTransaction();
+            transaction.replace(R.id.frame_layout_lux_meter, LuxMeterFragmentData.newInstance());
+            transaction.commit();
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+    }
+
+    @SuppressLint("ResourceType")
+    public void howToConnectDialog(String title, String intro, int iconID, String desc) {
+        try {
+            final AlertDialog.Builder builder = new AlertDialog.Builder(this);
+            LayoutInflater inflater = getLayoutInflater();
+            View dialogView = inflater.inflate(R.layout.custom_dialog_box, null);
+            builder.setView(dialogView);
+            builder.setTitle(title);
+            final TextView luxMeterGuideText = (TextView) dialogView.findViewById(R.id.custom_dialog_text);
+            final TextView luxMeterGuideDesc = (TextView) dialogView.findViewById(R.id.description_text);
+            final ImageView bh1750Schematic = (ImageView) dialogView.findViewById(R.id.custom_dialog_schematic);
+            final CheckBox doNotShowDialog = (CheckBox) dialogView.findViewById(R.id.toggle_show_again);
+            final Button dismisButton = (Button) dialogView.findViewById(R.id.dismiss_button);
+            luxMeterGuideText.setText(intro);
+            bh1750Schematic.setImageResource(iconID);
+            luxMeterGuideDesc.setText(desc);
+
+            final SharedPreferences sharedPreferences = getSharedPreferences(TAG, 0);
+            final AlertDialog dialog = builder.create();
+            Boolean skipDialog = sharedPreferences.getBoolean("skipDialog", false);
```

Comment:
I was going through all PRs related to dialog boxes and it seems like everyone's using too abstract variable name for shared preference boolean. Make this instrument specific. `settings.getBoolean("<skipLuxmeterMessage>", false);`
I'll request this change in other PRs as well.